src/ol_openedx_git_auto_export/ol_openedx_git_auto_export/tasks.py

```diff
@@ -20,14 +38,105 @@ def async_export_to_git(course_key_string, user=None):
             "Starting async course content export to git (course id: %s)",
             course_module.id,
         )
-        export_to_git(course_module.id, course_module.giturl, user=user)
+        course_repo = CourseGithubRepository.objects.get(course_id=course_key_string)
```

Comment:
We are overlooking the flag here and exporting in every case. Also, we should add appropriate logs.
```suggestion
        course_repo = CourseGithubRepository.objects.get(course_id=course_key_string, is_export_enabled=True)
```